src/analysis/classic/hunter/modules/features/Abilities.ts

```diff
@@ -0,0 +1,106 @@
+import SPELLS from 'common/SPELLS/classic/hunter';
+import CoreAbilities from 'parser/core/modules/Abilities';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import RACIALS from 'common/SPELLS/classic/racials';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    return [
+      // SPELLS ADDED HERE ARE DISPLAYED ON THE STATISTICS TAB
+      // LIST ALL SPELLS THAT COULD BE CAST DURING COMBAT BY THIS SPEC
```

Comment:
When running the site on local, review the console output in your browser for missing Abilities.

Reply:
I am doing that (from the docker console) and using it for compile errors/warnings. I didnt see anything. Is there another console besides the docker stdout ?

Reply:
When you run the site and open it in Chrome -> inspect -> console.
I didn't run your fork yet because there were a lot of issues to clean up first.